test/image-prepare

```diff
@@ -105,6 +114,18 @@ def build_install_package(dist_tar, image):
 
     args += ["--build", dist_tar]
 
+    # install the playground
+    args += ["--upload", os.path.join(BASE_DIR, "dist/playground") + ":/usr/local/share/cockpit/"]
+
+    # install mock pam module; we don't want to ship it as package, so fish it out of the build root
```

Comment:
I assume this is not tested in Testing Farm?

It feels a bit ooof, but I guess we can't install this in /usr/local? Because PAM and security?

Reply:
Right, the tests which use this are skipped as they don't test external APIs. We *could* build and install it there (even in /usr/lib) with a few extra build deps and shell code, but TBH I don't think it's worth the effort.

Reply:
I tested installing into /usr/local/lib64/security/mock-pam-conv-mod.so and PAM doesn't look there, I'm afraid. That's a bummer..

Reply:
Confirmed.